src/components/FloatingHelper/FloatingHelper.tsx

```diff
@@ -28,24 +38,43 @@ const FloatingHelperBO = withStylable<PopoverProps, {}>(
 );
 
 export const FloatingHelper: React.SFC<FloatingHelperProps> = props => {
-  const {children, content, ...rest} = props;
+  const {children, content, width, ...rest} = props;
+  const contentWidth = (typeof width) === 'number' ? `${width}px` : width;
+
   return (
     <FloatingHelperBO
       {...rest}
       showArrow
     >
       <Popover.Element>
           {children}
-        </Popover.Element>
-        <Popover.Content>
+      </Popover.Element>
+      <Popover.Content>
+        <div data-hook={DataHooks.contentWrapper} style={{width: contentWidth}}>
+          {props.withCloseButton && (
+            <Button
+              data-hook={DataHooks.closeButton}
+              className={classnames(style.closeButton, style.closeButtonWhiteSecondary)}
+            >
+              <CloseIcon className={classnames(style.closeIcon)}/>
```

Comment:
drop the classnames function if you have only one class

Reply:
done